tests/integration/tier1/destructive/one-kernel-scenario/test_one_kernel_scenario.py

```diff
@@ -42,10 +42,6 @@ def test_one_kernel_scenario(shell, convert2rhel, one_kernel):
     """TODO(r0x0d) better description and function name"""
 
     if os.environ["TMT_REBOOT_COUNT"] == "2":
-        # We need to skip check for collected rhsm custom facts after the conversion
-        # due to disabled submgr, thus adding envar
-        submgr_disabled_var = "SUBMGR_DISABLED_SKIP_CHECK_RHSM_CUSTOM_FACTS=1"
-        shell(f"echo '{submgr_disabled_var}' >> /etc/profile")
```

Comment:
I'm not sure, we can remove this one as it write the variable into the file

Reply:
Seems to be working just fine http://artifacts.osci.redhat.com/testing-farm/7a0b0d82-1b5d-40c2-b7af-15d0c8b460ae/work-one_kernel_scenario9akcym3c/log.txt

Reply:
Alright, cool!